slyd/app/components/annotations-plugin/component.js

```diff
@@ -132,10 +132,10 @@ export default Ember.Component.extend({
         },
 
         ignoreBeneath: function(_, value, index) {
-            var ignore = this.get('ignores').get(index),
+            var ignore = this.get('ignores').objectAt(index),
                 ignoreData = this.get('alldata').findBy('tagid', ignore.tagid);
-            ignore.ignoreBeneath = value;
-            ignoreData.ignoreBeneath = value;
+            ignore.set('ignoreBeneath', value);
+            ignoreData['ignore_beneath'] = value;
```

Comment:
.set isn't defined here because the elements of alldata aren't ember objects but regular javascript objects. I tried to find where alldata was being set and saw that it was activeExtractionTool.data.extracts which is set as this.get('extractionTools.' + tool_name) in controllers/template.js and that finally seems to map to model. It seems model is set before enableExtractionTool and thus is never set as this.get('annotationsStore').findAll() which would be ember objects. I couldn't find where model could be set before that function call. Forcing it to set alldata to this.get('annotationsStore').findAll() breaks quite a bit of code.